docs/mkdocs.yml

```diff
@@ -42,7 +43,9 @@ nav:
       - Terms of Use Setup: guides/providing-terms.md
   - Configuration: configuration.md
   - References:
-    - 'HedgeDoc Flavored Markdown': references/hfm.md
+      - 'HedgeDoc Flavored Markdown': references/hfm.md
+      - 'Slide options': references/slide-options.md
+      - 'URL scheme': references/url-scheme.md
```

Comment:
These should use Title Case, as the other link names

Reply:
fixed